paper.qmd

```diff
@@ -597,6 +604,53 @@ knitr::include_graphics("images/fig5.png")
+From the outset of the project, we aimed to follow @coppola2022's advocacy for a "human-centric approach in which personalization, purpose, ethics and inclusivity are key", rather than an 'off-the-shelf' product that failed to understand, let alone meet, local needs.
+As outlined in the overview of papers in the collection Inclusive and Collaborative Transport cited above, the 'digital transformation' taking place in transport planning (and many other sectors) has the potential to increase existing inequalities [@coppola2022].
+A wide range of methods and measure can be taken to ensure that digital tools are inclusive \[...\].
+
+<!-- Details from final workshop:
+
+Can you please join me and my colleagues from TII and ITS Leeds for 1 hour on Thursday the 7th December at 12:00 – 13:00 for a workshop and demonstration of the open source CRUSE tool) which is used primarily to highlight the cycle friendliness of roads in Ireland, avail of cycle scenario/trip data and highlight areas to target for active travel investment. This tool is free to use for transport planners or member of the general public and can be found here: CRUSE . We have now added recreational cycling trips to the CRUSE tool and would welcome the chance to present this work and ask for feedback.
+
+Please see below a draft agenda for the workshop till 1pm:
+
+-   Introduction (5 min)
+-   Changes to CRUSE: extension work (15 min)
+-   Testing new website (10 min)
+-   Feedback, Q&A + discussion (30 min)
+    - General feedback on new site
+    - Feedback on the recreational outputs
+    - Feedback on the tourism network -->
+
+
+<!-- – What are main issues you face in
+understanding infrastructure?, e.g.
+– What is happening at the moment?
+– Where to invest in the future?
+– What will change?
+– How will planning vary depending on trip purpose?
+– What sort of scenarios interested in testing?
+– How useful is having different networks for
+planning infrastructure and/or more detail,
+e.g. there could be an option for variation in
+access for professionals and public (the latter
+remaining simple) -->
+
+Due to time and skill constraints in the team developing the CRUSE tool (we lacked researchers with a background in qualitative methods), we opted for the well-tested and cost-effective approach of holding workshops with stakeholders in case study area, an approach that is documented with reference to participatory development of digital tools for supporting energy transitions [@hewitt2020].
+We held a total of three workshops over the course of the project and one open-ended live demo during a 2-day event in Sligo in which we collected feedback from dozens of people.
+Other than the live demo, all of the workshops took place online.
+The first two workshops took place in May 2022, in which key staff involved in cycle network planning in Kildare and Limerick counties were presented the tool.
```

Comment:
Rephrase this sentence to “The first two workshops took place in May 2022, during which the tool was introduced to key staff involved in cycle network planning in Kildare and Limerick counties”.